### ISSUE TITLE
[BUG] int64 for src/dst fails number_of_vertices

### ISSUE THREAD

**Issue by geoHeil** [rapidsai/cugraph#752]
When using int32 for src/dst everything is fine. But int64:
```
G.number_of_vertices()
RuntimeError: CUGRAPH failure at: /conda/conda-bld/libcugraph_1580841414814/work/cpp/src/structure/cugraph.cu:256: Unsupported data type
```

this is only to be reproduced on my real data, not on the karate example graph from cugraph github

**Comment by BradReesWork** [comment]
@geoHeil:  We currently do not support int64 for vertex IDs :(   That is a major item that we plan on addressing in the 0.14 release

**Comment by geoHeil** [comment]
no problem. Interestingly degree calculation works fine with int64

**Comment by Iroy30** [comment]
This is going to be fixed by PR #747
